### DISCUSSION TITLE
Apple Vision Pro support

### DISCUSSION THREAD

**Discussion by felipecocco** [f1multiviewer/issue-tracker#475] [Feature requests]
Needless to say, the killer environment for something like Multiviewer would be the AVP. Even without some futuristic 3d modeling of the track -- just pure ability to spin up the different windows, with some ability to sync between them in virtual space, would handily make Multiviewer the best app on Vision Pro.

**Comment by LapsTimeOFF** [comment]
The Apple Vision Pro is currently not in our plans as the framework we use is not available on those platforms.
